kubernetes-dashboard-auth.yaml

```diff
@@ -51,6 +56,37 @@ test:
     - name: Check version in logs
       runs: |
         grep "${{package.version}}" dashboard-auth.log || exit 1
+    - name: "Start dashboard-auth service"
+      uses: test/daemon-check-output
+      with:
+        setup: |
+          if nc -z localhost 8000; then
+            echo "Port 8000 is already in use"
+            exit 1
+          fi
+        start: |
+          dashboard-auth \
+            --address=0.0.0.0 \
+            --port=8000 \
+            --apiserver-host="https://127.0.0.1:32764" \
+            --kubeconfig=/root/.kube/config \
+            --apiserver-skip-tls-verify \
+            -v=4
+        timeout: 30
+        expected_output: |
+          "Starting Kubernetes Dashboard Auth" version="${{package.version}}"
+          "Listening and serving insecurely on" address="0.0.0.0:8000"
+        post: |
+          wait-for-it -t 30 localhost:8000 || exit 1
+
+          # Check Kubernetes API health
+          for endpoint in / /readyz /livez /openapi/v2; do
+          echo "Checking $endpoint..."
+          kubectl get --raw "$endpoint" || {
+            echo "Failed to query $endpoint"
+            exit 1
+          }
+          done
```

Comment:
Not sure if we need this as we are talking with API server, not with the dashboard.